src/main/java/Avocado.java

```diff
@@ -0,0 +1,31 @@
+import java.util.Scanner;
+public class Avocado {
+    public static void main(String[] args) {
+        System.out.println("Hello from avocado");
+        System.out.println("How can I help you ?");
+        System.out.println("See you!");
+        String[] list = new String[100];
```

Comment:
It would be better to define 100 as a constant, for example `static final int LIST_SIZE = 100;`